setup.py

```diff
@@ -34,35 +34,41 @@
     if os.path.exists('MANIFEST'):
         os.remove('MANIFEST')
 
-    setup(name=DISTNAME,
-          maintainer=MAINTAINER,
-          include_package_data=True,
-          maintainer_email=MAINTAINER_EMAIL,
-          description=DESCRIPTION,
-          license=LICENSE,
-          url=URL,
-          version=VERSION,
-          download_url=DOWNLOAD_URL,
-          long_description=open('README.md').read(),
-          zip_safe=False,  # the package can run out of an .egg file
-          classifiers=['Intended Audience :: Science/Research',
-                       'Intended Audience :: Developers',
-                       'License :: OSI Approved',
-                       'Programming Language :: Python',
-                       'Topic :: Software Development',
-                       'Topic :: Scientific/Engineering',
-                       'Operating System :: Microsoft :: Windows',
-                       'Operating System :: POSIX',
-                       'Operating System :: Unix',
-                       'Operating System :: MacOS'],
-          platforms='any',
-          packages=find_packages(),
-          entry_points={
-              'console_scripts': [
-                  'ramp_test_submission='
-                  'rampwf.utils.command_line:ramp_test_submission',
-                  'ramp_test_notebook='
-                  'rampwf.utils.command_line:ramp_test_notebook',
-                  'ramp_convert_notebook='
-                  'rampwf.utils.command_line:ramp_convert_notebook',
-              ]})
+    setup(
+        name=DISTNAME,
+        maintainer=MAINTAINER,
+        include_package_data=True,
+        maintainer_email=MAINTAINER_EMAIL,
+        description=DESCRIPTION,
+        license=LICENSE,
+        url=URL,
+        version=VERSION,
+        download_url=DOWNLOAD_URL,
+        long_description=open('README.md').read(),
+        zip_safe=False,  # the package can run out of an .egg file
+        classifiers=[
+            'Intended Audience :: Science/Research',
+            'Intended Audience :: Developers',
+            'License :: OSI Approved',
+            'Programming Language :: Python',
+            'Topic :: Software Development',
+            'Topic :: Scientific/Engineering',
+            'Operating System :: Microsoft :: Windows',
+            'Operating System :: POSIX',
+            'Operating System :: Unix',
+            'Operating System :: MacOS'],
+        install_requires=[
+            'numpy>=1.13',
+            'scipy>=0.19',
+            'scikit-learn>=0.18'],
```

Comment:
although pandas is not a strict requirement (it is currently not directly imported), it is still used a lot in the code (code that assumes pandas data structures are used), so therefore would add it here

Reply:
fair enough

Reply:
what should be the version required ?

Reply:
Hmm, not sure. Let's maybe take 0.19 for now ? 
(actually, we should in principle also test for all minimal dependencies, but can be for other PR)